packages/jest-environment/src/index.ts

```diff
@@ -300,12 +300,19 @@ export interface Jest {
    *
    * `waitBeforeRetry` is the number of milliseconds to wait before retrying
    *
+   * `immediately` is the flag to retry the failed tests immediately after
+   *  failure
+   *
    * @remarks
    * Only available with `jest-circus` runner.
    */
   retryTimes(
     numRetries: number,
-    options?: {logErrorsBeforeRetry?: boolean; waitBeforeRetry?: number},
+    options?: {
+      logErrorsBeforeRetry?: boolean;
+      waitBeforeRetry?: number;
+      immediately?: boolean;
```

Comment:
Would be nice to keep alphabetical order.

Reply:
Fixed